blueprint/index.md

```diff
@@ -1,305 +1,345 @@
+## Implementing a support platform that delivers exceptional customer experience
+The Genesys Cloud Developer Blueprint provides a simple example of implementing Limitless GigCX on Genesys Cloud [https://www.limitlesstech.com/genesys/](https://www.limitlesstech.com/genesys/ "Goes to the GigCX with Limitless and Genesys page") to manage and deliver expert customer support.
 
-- Limitless GigCX
-- Connecting To Limitless SmartCrowd
-- The Data Actions
-- The Architect Flows:
-  - Open messaging
-  - Chat (This flow is compatible with web chat v1.1 & web chat v2)
-  - Email
-- Additional Considerations
-## **Limitless GigCX**
-Via Limitless GigCX your organization can quickly build a qualified crowd of expert product users to resolve customer support tickets. Experts are available 24/7, in any language, to deliver amazing support at a significantly lower cost to serve. You can find out more about Limitless and GigCX at: <https://www.limitlesstech.com/>
+This blueprint demonstrates how to:
 
-This blueprint brings the power of Limitless and GigCX to Genesys Cloud architect flows. Using these flows, open messaging, chat and email channels in Genesys Cloud can be connected to Limitless APIs to bring GigCX Experts into the conversation via the SmartCrowd platform. 
+- Integrate Genesys Cloud and the Limitless GigCX SmartCrowd platform
+- Use Genesys Cloud open messaging and chats
+- Connect Limitless GigCX to your email channels
+- Route a conversation to an expert crowd and facilitate the conversation between customer and GigCX expert
+- Route to the appropriate Genesys Cloud agent if the expert crowd is unable to help
+- Manage the lifecycle and status events from the SmartCrowd platform
 
-The flow performs the following functions:
+## Solution components
 
-- Routing of conversations to Expert Crowd
-- Integration between Genesys Cloud and the Limitless SmartCrowd platform
-- Facilitating the conversation between Customer and GigCX Expert
-- Manages the lifecycle/status events coming from the SmartCrowd platform and routes accordingly within Genesys
-- Brings a Genesys Agent into the conversation if the Expert Crowd is unable to help the Customer
+**Limitless GigCX** - A product your organization can use to quickly build a qualifier crowd of exports to resolve customer support tickets. The expert is available 24/7, in any language, to deliver amazing support at a significantly lower cost to serve. You can find out more about Limitless GigCX at: <https://www.limitlesstech.com/>
 
-A high-level representation of the implementation can be seen in the diagram.
+**Architect Flows** - A flow in Architect, a drag and drop web-based design tool, dictates how Genesys Cloud handles inbound or outbound interactions. 
 
-![](images/001.png) 
+## Prerequisites
 
-You can learn more about Limitless and Genesys at: <https://www.limitlesstech.com/genesys/>
+### Specialized knowledge
+- Compatible with web chat v1.1 & web chat v2
```

Comment:
Experience configuring and installing Genesys Cloud web chat v1.1 or Genesys Cloud v2